_data/navigation.yml

```diff
@@ -44,81 +37,16 @@
   - title: "Online Workshop Recommendations"
     url: "https://carpentries.org/online-workshop-recommendations/"
 
-- title: Learn
-  url: "/learn/"
-  side: left
-  dropdown:
-  - title: "Our Workshops"
-    url: "/workshops/"
-  - title: "Community Lessons"
-    url: "/community-lessons/"
-
-  # - title: "Our Curricula"
```

Comment:
Isn't it a bit early to remove all these? I'd prefer to leave the breadcrumbs to make it easier for my future self. In an ideal we will be able to list our past and future workshops via Amy

Reply:
No. We can design versions of our own using hand-curated documents, by AMY is not ours to use.

Reply:
We are not (and never were) using AMY. I hand curated the json file that AMY _could_ provide. I still believe that this is the right approach, whether we use AMY or something else. If we use json then we are in synch with upstream and ready to transition.

I am specifically talking about the comments here though, we do  aspire to use AMY for these at some point, then these comments become useful.

Reply:
Ahh, sorry, I was too hasty & did not look closely enough. Since this page used "HPC Incubator" and "HPC Lab", and neither of those things has been discussed elsewhere, I assumed this was a Carpentries page with some text search-and-replaced. Turns out, it's not! I don't mind restoring it, but just like "Our Workshops" points to just "/workshops/", the summary of HPC Carpentry lessons should be found at "/lessons/", and the only distinction to be made is whether they reside in The Carpentries Incubator or under HPC Carpentry. If a third section for CodeRefinery is to be added, we should plan out the appropriate way to refer to that third-party organization and their materials.

Reply:
Restored, with modifications, by 333a4a9e.